crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -138,11 +138,15 @@ def adminrepocheck
             }
           end
 
-          unless zypper_stream["prompt"].nil?
+          prompt = zypper_stream["prompt"]
+          unless prompt.nil?
+            # keep only first prompt for easier formatting
+            prompt = prompt.first if prompt.is_a?(Array)
```

Comment:
can't we just concatenate the prompts? the formatting will be already kind of ugly i assume?